docs/cody/clients/cody-with-sourcegraph.mdx

```diff
@@ -33,7 +33,7 @@ Sourcegraph.com users with Cody **Free** and **Pro** can choose from a list of s
 
 ![llm-select-web](https://storage.googleapis.com/sourcegraph-assets/Docs/llm-select-web-0724.jpg)
 
-Users on an Enterprise Sourcegraph instance do not have the option to choose an LLM model. Their site admin will configure the default LLM model for chat.
+Users on an Enterprise Sourcegraph instance do not have the option to choose an LLM model. Their site admin will configure the default LLM model for chat. However, Enterprise users with the new [model configuration](/cody/clients/model-configuration) can use the LLM selection dropdown to choose a chat model.
```

Comment:
This is true, right, @aramaraju ?

Reply:
We can capture that in Early Access to be super clear on that statement, but functionally accurate